runtime/vm/createramclass.cpp

```diff
@@ -1530,16 +1530,24 @@ loadSuperClassAndInterfaces(J9VMThread *vmThread, J9ClassLoader *classLoader, J9
 			}
 
 			/* ensure that the superclass isn't an interface or final */
-			if ((superclass->romClass->modifiers & J9_JAVA_FINAL) != 0) {
+			if (J9_ARE_ALL_BITS_SET(superclass->romClass->modifiers, J9AccFinal)) {
```

Comment:
you can cache `(superclass->romClass->modifiers)` and update all the other places that it is used